### PR TITLE
Add article about CNI

### PR DESCRIPTION
**Pull request by astraw99**
Add article about CNI.

### CODE REVIEW

File: articles/K8s 系列(六) - 浅谈 CNI.md [hunk 1]
```diff
@@ -1 +1,290 @@
-## 更新中...
\ No newline at end of file
+## 1. 概述
+进入 K8s 的世界，会发现有很多方便扩展的 Interface，包括 CNI, CSI, CRI 等，将这些接口抽象出来，是为了更好的提供开放、扩展、规范等能力。
+
+K8s 网络模型采用 CNI(Container Network Interface, 容器网络接口) 协议，只要提供一个标准的接口，就能为同样满足该协议的所有容器平台提供网络功能。
+
+CNI 是 CoreOS 提出的一种容器网络规范，目前已被 Apache Mesos、Cloud Foundry、Kubernetes、Kurma、rkt 等众多开源项目所采用，同时也是一个 CNCF(Cloud Native Computing Foundation) 项目。可以预见，CNI 将会成为未来容器网络的标准。
+
+本文将从 kubelet 启动、Pod 创建/删除、Docker 创建/删除 Container、CNI RPC 调用、容器网络配置等核心流程，对 CSI 实现机制进行了解析。
```
Comment (NoicFank) [inline articles/K8s 系列(六) - 浅谈 CNI.md:8]: 对CSI -> 对CNI

File: articles/K8s 系列(六) - 浅谈 CNI.md [hunk 2]
```diff
@@ -1 +1,290 @@
...
+
+	return resp, nil
+}
+```
+
+流程图小结如下：
+
+![K8s-CNI-flow](../images/CNI/K8s-CNI-flow.png)
+
+> 根据社区讨论(https://kubernetes.io/blog/2020/12/02/dockershim-faq/)，dockershim 相关代码将会在 2021 底左右移出 K8s 主干代码，之后将统一使用 CRI(Container Runtime Interface, 容器运行时接口) 进行容器生命周期管理。
```
Comment (NoicFank) [inline articles/K8s 系列(六) - 浅谈 CNI.md:265]: 链接显示bug，去除最后的/。(https://kubernetes.io/blog/2020/12/02/dockershim-faq/) -> (https://kubernetes.io/blog/2020/12/02/dockershim-faq)

File: articles/K8s 系列(六) - 浅谈 CNI.md (hunk 1 above)
Comment (astraw99) [reply articles/K8s 系列(六) - 浅谈 CNI.md]: Fixed.

File: articles/K8s 系列(六) - 浅谈 CNI.md (hunk 2 above)
Comment (astraw99) [reply articles/K8s 系列(六) - 浅谈 CNI.md]: Fixed.

**Review by sxllwx — APPROVED** [review]
lgtm

**Pull request merged**